### PR TITLE
Adding catalog icon + updated catalog attributes

### PR DESCRIPTION
**Pull request by wjimenez5271**

### CODE REVIEW

File: templates/iron-functions/config.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 name: IronFunctions
 description: |
-  Serverless platform using Docker
+  Serverless platform using Docker (https://github.com/iron-io/functions)
 version: v0.0.1
```
Comment (rawmind0) [inline templates/iron-functions/config.yml]: Could you please set correct default version?? 
```
-  version: v0.0.1
+  version: v0.0.2
```

**Comment by rawmind0** [comment]
Hi @wjimenez5271 , thanks for the PR... :)

drone checks are failing due to package default version. Please take a look at change request.

File: templates/iron-functions/config.yml (hunk 1 above)
Comment (wjimenez5271) [inline templates/iron-functions/config.yml]: ah, good catch, sorry still learning the structure of these

**Comment by rawmind0** [comment]
Thanks @wjimenez5271 ...

LGTM

**Pull request merged**